src/client/session.js

```diff
@@ -0,0 +1,14 @@
+import store from 'store2';
+import { Random } from 'meteor/random';
+
+export function getUserHex() {
+  if (!store.has('user')) {
+    store.set('user', Random.hexString(32));

```

Comment:
I see that the hex is generated on the client side. I know that it is very unlikely for two random hexes to be the same, but wouldn't it be more idiomatic to make a GET request to the server and have the server generate a hash that is collision-checked?

If you want to proceed with this, I am fine, but I want to make sure we are aware of it.